akka-bbb-apps/src/main/scala/org/bigbluebutton/core/util/jhotdraw/Bezier.java

```diff
@@ -21,21 +21,6 @@
 import java.awt.geom.*;
 import java.util.*;
 
-/**
- * Provides algorithms for fitting Bezier curves to a set of digitized points.
- * <p>
- * Source:<br>
- * Phoenix: An Interactive Curve Design System Based on the Automatic Fitting
- * of Hand-Sketched Curves.<br>
- * © Copyright by Philip J. Schneider 1988.<br>
- * A thesis submitted in partial fulfillment of the requirements for the degree
- * of Master of Science, University of Washington.
- * <p>
- * http://autotrace.sourceforge.net/Interactive_Curve_Design.ps.gz
- *
- * @author Werner Randelshofer
- * @version $Id$
- */
```

Comment:
I'm going to guess that these blocks were auto-removed by some script on your computer. Could please add them back